manheim_cloudmapper/ses/report.py

```diff
@@ -0,0 +1,179 @@
+import os
+import datetime
+import logging
+from premailer import transform
+from .ses import SES
+
+logger = logging.getLogger(__name__)
+
+
+class Report():
+
+    # Base path for CSS stylings, default for cloudmapper
+    BASE_PATH = '/opt/manheim_cloudmapper/web/css'
+
+    def __init__(self,
+                 report_source=(
+                     '/opt/manheim_cloudmapper/web/account-data/report.html'
+                 ),
+                 account_name=None, sender=None, recipient=None,
+                 region=None, ses_enabled=None):
+        """
+        Initialize the Cloudmapper report. Sets variables for email.
+
+        :param account_name: AWS account name
+        :type account_name: str
+        :param sender: Email address registered to SES that will send emails
+        :type sender: str
+        :param recipient: Email address to send emails to
+        :type recipient: str
+        :param region: AWS Region
+        :type region: str
+        :param ses_enabled: wheter the SES email functionality is enabled
+        :type ses_enabled: str
```

Comment:
We should never use a string to represent a boolean state. Wherever we're initially getting this string, it should be converted to a boolean, and this should take a boolean.